### DISCUSSION TITLE
World seed

### DISCUSSION THREAD

**Discussion by object-Object** [HexxyCraft/modpack#11] [General]
Should the world seed be public?

**Comment by Robotgiggle** [comment]
I'd suggest keeping the seed private, to prevent people from using structure finders or instantly getting all the great spells.

**Comment by maraxin** [comment]
It's up to you, if you want this server to be more chill, you can make it public, but if you want it to make more difficult, then you can keep it private. I think its just about how server goes

**Comment by LlamaSageR** [comment]
The seed should be private. There's no utility it provides except for people trying to exploit it. While it is possible to find it out regardless through funny techniques, there's no real point to it in my opinion.

**Comment by walksanatora** [comment]
> I'd suggest keeping the seed private, to prevent people from using structure finders or instantly getting all the great spells.

we (plan on) lootr
so loot is per-player anyways
but eitherway private seed is better

**Comment by object-Object** [comment]
Closing as a consensus seems to have been reached. The world seed will remain private.
